manim/mobject/geometry.py

```diff
@@ -812,7 +813,27 @@ def pointify(self, mob_or_point, direction=None):
                 return mob.get_boundary_point(direction)
         return np.array(mob_or_point)
 
-    def put_start_and_end_on(self, start, end):
+    def put_start_and_end_on(self, start: Sequence[float], end: Sequence[float]):
+        """
```

Comment:
Write the description as:
Gets starting point and an ending point for a `Mobject`.

It can be used to shift animations to those particular points or just some objects too.

Reply:
> Gets starting point and an ending point for a Mobject.
No, fist of all this is not a method of Mobjects, but of Line.
Second, it does not get these points, but it sets these points.